src/flint.h

```diff
@@ -12,14 +12,21 @@
 #ifndef FLINT_H
 #define FLINT_H
 
+#ifndef FLINT_NOSTDIO
+# include <stdio.h>
+#endif
+
+#ifndef FLINT_NOSTDARG
+# include <stdarg.h>
+#endif
+
 #if !defined(_MSC_VER)
 #include <sys/param.h> /* for BSD define */
 #endif
-#include <stdio.h>
-#include <stdarg.h>
 #include <gmp.h>
 #include "limits.h"
 #include "longlong.h"
+#include "flint_types.h"
```

Comment:
Isn't the inclusion of an extra header file going to slow down compilation?

Reply:
Yeah, probably. However, not by much.